ui/selectors/selectors.test.js

```diff
@@ -1043,6 +1074,7 @@ describe('Selectors', () => {
         balance: '0x0',
         pinned: true,
         hidden: false,
+        active: false,
```

Comment:
I have updated the updated the test, by checking the active false. For the connected one, it's true else false